pkg/openapi/getschema.go

```diff
@@ -0,0 +1,277 @@
+package openapi
+
+import (
+	"encoding/json"
+	"fmt"
+	"slices"
+	"strings"
+
+	"github.com/getkin/kin-openapi/openapi3"
+)
+
+type Parameter struct {
+	Name    string `json:"name"`
+	Style   string `json:"style"`
+	Explode *bool  `json:"explode"`
+}
+
+type OperationInfo struct {
+	Server          string           `json:"server"`
+	Path            string           `json:"path"`
+	Method          string           `json:"method"`
+	BodyContentMIME string           `json:"bodyContentMIME"`
+	SecurityInfos   [][]SecurityInfo `json:"securityInfos"`
+	QueryParams     []Parameter      `json:"queryParameters"`
+	PathParams      []Parameter      `json:"pathParameters"`
+	HeaderParams    []Parameter      `json:"headerParameters"`
+	CookieParams    []Parameter      `json:"cookieParameters"`
+}
+
+var (
+	SupportedMIMETypes     = []string{"application/json", "application/x-www-form-urlencoded", "multipart/form-data"}
+	SupportedSecurityTypes = []string{"apiKey", "http"}
```

Comment:
nit: my feeling is that it is always better to have non-const package level variables by unexported and use a getter.

Reply:
Done